### PR TITLE
Update chart-releaser to fix cache-control when storing index file

### LINKED ISSUE #1477: Update chart-releaser to fix cache-control when storing index file
**What would you like to be added**:

From this issue https://github.com/pipe-cd/pipe/issues/1418
> It seems the configured metadata will be reset whenever the file content is updated.

So we have to configure the metadata for the index file whenever storing it to GCS.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1477

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: dockers/chart-releaser/main.go [hunk 1]
```diff
@@ -131,13 +132,13 @@ func main() {
 			return nil
 		}
 		log.Printf("Start uploading package %s...", info.Name())
-		return storeFile(ctx, client, path)
+		return storeFile(ctx, client, path, false)
```
Comment (khanhtc1202) [inline dockers/chart-releaser/main.go:135]: nit, add note represents `we do not disable cache in this case since ...` help the future debugging process 👍

Comment (nghialv) [reply dockers/chart-releaser/main.go:135]: Sure.

Comment (nghialv) [reply dockers/chart-releaser/main.go:136]: Added the comment.

**Comment by khanhtc1202** [comment]
🚀 
/lgtm

**Review by nakabonne — APPROVED** [review]

**Pull request merged**